src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumerableToDispatchMarshaler.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    public class EnumerableToDispatchMarshaler : ICustomMarshaler
+    {
+        private static readonly EnumerableToDispatchMarshaler s_enumerableToDispatchMarshaler = new EnumerableToDispatchMarshaler();
+
+        public static ICustomMarshaler GetInstance(string cookie) => s_enumerableToDispatchMarshaler;
+
+        private EnumerableToDispatchMarshaler()
+        {}
+
+        public void CleanUpManagedData(object ManagedObj)
+        {
+        }
+
+        public void CleanUpNativeData(IntPtr pNativeData)
+        {
+            Marshal.Release(pNativeData);
+        }
+
+        public int GetNativeDataSize()
+        {
+            return -1;
+        }
+
+        public IntPtr MarshalManagedToNative(object ManagedObj)
```

Comment:
variables should use camelCasing.

Reply:
I'm matching the declaration of `System.Runtime.InteropServices.ICustomMarshaler`.

https://github.com/dotnet/corefx/blob/8afa2cdfc6c16b381ccab80c11bf5349fd5b1c21/src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs#L445

I can change it in these implementations if you want.

Reply:
Ah I see. Then it's fine as is.